[Mon Nov 11 16:30:23.246251 2019] [autoindex:error] [pid 21294] [client 202.146.192.143:5587] AH01276: Cannot serve directory /home/pyb78/pyb78.specind.net/: No matching DirectoryIndex (index.py) found, and server-generated directory index forbidden by Options directive
[Wed Nov 13 11:49:52.388196 2019] [autoindex:error] [pid 2082] [client 192.168.15.165:53532] AH01276: Cannot serve directory /home/pyb78/pyb78.specind.net/: No matching DirectoryIndex (index.py) found, and server-generated directory index forbidden by Options directive
[Tue Nov 19 13:59:38.977873 2019] [cgi:error] [pid 6537] [client 223.228.177.190:64072] AH01215: (13)Permission denied: exec of '/home/pyb78/pyb78.specind.net/register.py' failed: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Tue Nov 19 13:59:38.978885 2019] [cgi:error] [pid 6537] [client 223.228.177.190:64072] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 10:18:07.283358 2019] [autoindex:error] [pid 10131] [client 223.228.136.70:6929] AH01276: Cannot serve directory /home/pyb78/pyb78.specind.net/: No matching DirectoryIndex (index.py) found, and server-generated directory index forbidden by Options directive
[Wed Nov 20 10:23:12.547501 2019] [http:error] [pid 10303] [client 223.228.136.70:50460] AH02429: Response header name '<!--' contains invalid characters, aborting request, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 10:30:52.983131 2019] [http:error] [pid 10559] [client 223.228.136.70:57313] AH02429: Response header name '<!--' contains invalid characters, aborting request, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 10:42:14.257574 2019] [http:error] [pid 11059] [client 223.228.136.70:54648] AH02429: Response header name '<!--' contains invalid characters, aborting request, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 10:44:32.583793 2019] [http:error] [pid 11158] [client 223.228.136.70:29839] AH02429: Response header name '<!--' contains invalid characters, aborting request, referer: http://pyb78.specind.net/registration.html?
[Wed Nov 20 10:50:14.432949 2019] [http:error] [pid 11355] [client 223.228.136.70:39838] AH02429: Response header name '<!--' contains invalid characters, aborting request, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:00:00.017920 2019] [autoindex:error] [pid 11658] [client 223.228.136.70:13178] AH01276: Cannot serve directory /home/pyb78/pyb78.specind.net/: No matching DirectoryIndex (index.py) found, and server-generated directory index forbidden by Options directive
[Wed Nov 20 11:37:45.979382 2019] [cgi:error] [pid 12988] [client 223.228.136.70:51842] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 79: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:37:45.979598 2019] [cgi:error] [pid 12988] [client 223.228.136.70:51842] AH01215:     sql1 = 'INSERT INTO payment_page(id,user_id,customer_profile_id,payment_profile_id,card_no)VALUES("%d","%d","%d","%s","%d")' %(id,user_id,customer_profile_id,payment_profile_id,card_no): /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:37:45.979614 2019] [cgi:error] [pid 12988] [client 223.228.136.70:51842] AH01215:     ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:37:45.979643 2019] [cgi:error] [pid 12988] [client 223.228.136.70:51842] AH01215: IndentationError: unexpected indent: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:37:45.981479 2019] [cgi:error] [pid 12988] [client 223.228.136.70:51842] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:38:30.486904 2019] [cgi:error] [pid 13015] [client 223.228.136.70:23645] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 79: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:38:30.487109 2019] [cgi:error] [pid 13015] [client 223.228.136.70:23645] AH01215:     sql1 = 'INSERT INTO payment_page(id,user_id,customer_profile_id,payment_profile_id,card_no)VALUES("%d","%d","%d","%s","%d")' %(id,user_id,customer_profile_id,payment_profile_id,card_no): /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:38:30.487122 2019] [cgi:error] [pid 13015] [client 223.228.136.70:23645] AH01215:     ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:38:30.487151 2019] [cgi:error] [pid 13015] [client 223.228.136.70:23645] AH01215: IndentationError: unexpected indent: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:38:30.489042 2019] [cgi:error] [pid 13015] [client 223.228.136.70:23645] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:41:50.302538 2019] [cgi:error] [pid 13180] [client 223.228.136.70:44636] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 79: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:41:50.302726 2019] [cgi:error] [pid 13180] [client 223.228.136.70:44636] AH01215:     sql1 = 'INSERT INTO payment_page(user_id,customer_profile_id,payment_profile_id,card_no)VALUES(%d","%d","%s","%d")' %(user_id,customer_profile_id,payment_profile_id,card_no): /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:41:50.302739 2019] [cgi:error] [pid 13180] [client 223.228.136.70:44636] AH01215:     ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:41:50.302768 2019] [cgi:error] [pid 13180] [client 223.228.136.70:44636] AH01215: IndentationError: unexpected indent: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:41:50.304677 2019] [cgi:error] [pid 13180] [client 223.228.136.70:44636] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:44:46.857445 2019] [cgi:error] [pid 13260] [client 223.228.136.70:20384] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 79: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:44:46.857634 2019] [cgi:error] [pid 13260] [client 223.228.136.70:20384] AH01215:     sql1 = 'INSERT INTO payment_page(user_id,customer_profile_id,payment_profile_id,card_no)VALUES(%d","%d","%s","%d")' %(user_id,customer_profile_id,payment_profile_id,card_no): /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:44:46.857648 2019] [cgi:error] [pid 13260] [client 223.228.136.70:20384] AH01215:     ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:44:46.857677 2019] [cgi:error] [pid 13260] [client 223.228.136.70:20384] AH01215: IndentationError: unexpected indent: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:44:46.859602 2019] [cgi:error] [pid 13260] [client 223.228.136.70:20384] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:46:22.274855 2019] [cgi:error] [pid 13315] [client 223.228.136.70:64925] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 79: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:46:22.275036 2019] [cgi:error] [pid 13315] [client 223.228.136.70:64925] AH01215:     sql1 = 'INSERT INTO payment_page(user_id,customer_profile_id,payment_profile_id,card_no)VALUES(%d","%d","%s","%d")' %(user_id,customer_profile_id,payment_profile_id,card_no): /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:46:22.275056 2019] [cgi:error] [pid 13315] [client 223.228.136.70:64925] AH01215:     ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:46:22.275084 2019] [cgi:error] [pid 13315] [client 223.228.136.70:64925] AH01215: IndentationError: unexpected indent: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 11:46:22.276860 2019] [cgi:error] [pid 13315] [client 223.228.136.70:64925] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:05:26.863805 2019] [cgi:error] [pid 13936] [client 223.228.136.70:49596] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 79: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:05:26.863995 2019] [cgi:error] [pid 13936] [client 223.228.136.70:49596] AH01215:     sql1 = 'INSERT INTO payment_page(user_id,customer_profile_id,payment_profile_id,card_no)VALUES(%d","%d","%s","%d")' %(user_id,customer_profile_id,payment_profile_id,card_no): /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:05:26.864009 2019] [cgi:error] [pid 13936] [client 223.228.136.70:49596] AH01215:     ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:05:26.864039 2019] [cgi:error] [pid 13936] [client 223.228.136.70:49596] AH01215: IndentationError: unexpected indent: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:05:26.866001 2019] [cgi:error] [pid 13936] [client 223.228.136.70:49596] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:10:07.514104 2019] [cgi:error] [pid 14122] [client 223.228.136.70:15403] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 79: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:10:07.514242 2019] [cgi:error] [pid 14122] [client 223.228.136.70:15403] AH01215:     sql1 = 'INSERT INTO payment_page(username,customer_profile_id)VALUES("%s","%d")' %(Username,customer_profile_id): /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:10:07.514252 2019] [cgi:error] [pid 14122] [client 223.228.136.70:15403] AH01215:     ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:10:07.514280 2019] [cgi:error] [pid 14122] [client 223.228.136.70:15403] AH01215: IndentationError: unexpected indent: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:10:07.516004 2019] [cgi:error] [pid 14122] [client 223.228.136.70:15403] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:44:50.574913 2019] [cgi:error] [pid 16089] [client 223.228.136.70:64623] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 35: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:44:50.574990 2019] [cgi:error] [pid 16089] [client 223.228.136.70:64623] AH01215:     0import pymysql: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:44:50.575003 2019] [cgi:error] [pid 16089] [client 223.228.136.70:64623] AH01215:           ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:44:50.575025 2019] [cgi:error] [pid 16089] [client 223.228.136.70:64623] AH01215: SyntaxError: invalid syntax: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:44:50.576867 2019] [cgi:error] [pid 16089] [client 223.228.136.70:64623] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:52:58.517935 2019] [cgi:error] [pid 16344] [client 223.228.136.70:12940] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 35: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:52:58.518012 2019] [cgi:error] [pid 16344] [client 223.228.136.70:12940] AH01215:     0import pymysql: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:52:58.518025 2019] [cgi:error] [pid 16344] [client 223.228.136.70:12940] AH01215:           ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:52:58.518048 2019] [cgi:error] [pid 16344] [client 223.228.136.70:12940] AH01215: SyntaxError: invalid syntax: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 12:52:58.519857 2019] [cgi:error] [pid 16344] [client 223.228.136.70:12940] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 13:01:15.830817 2019] [cgi:error] [pid 16605] [client 223.228.136.70:24822] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 35: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 13:01:15.830900 2019] [cgi:error] [pid 16605] [client 223.228.136.70:24822] AH01215:     0import pymysql: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 13:01:15.830914 2019] [cgi:error] [pid 16605] [client 223.228.136.70:24822] AH01215:           ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 13:01:15.830939 2019] [cgi:error] [pid 16605] [client 223.228.136.70:24822] AH01215: SyntaxError: invalid syntax: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 13:01:15.832831 2019] [cgi:error] [pid 16605] [client 223.228.136.70:24822] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Wed Nov 20 14:53:29.822492 2019] [autoindex:error] [pid 20597] [client 223.228.136.70:40498] AH01276: Cannot serve directory /home/pyb78/pyb78.specind.net/: No matching DirectoryIndex (index.py) found, and server-generated directory index forbidden by Options directive
[Wed Nov 20 14:53:30.309413 2019] [autoindex:error] [pid 20597] [client 223.228.136.70:40498] AH01276: Cannot serve directory /home/pyb78/pyb78.specind.net/: No matching DirectoryIndex (index.py) found, and server-generated directory index forbidden by Options directive
[Wed Nov 20 15:30:21.992856 2019] [cgi:error] [pid 21996] [client 223.228.136.70:6088] AH01215: (13)Permission denied: exec of '/home/pyb78/pyb78.specind.net/success.py' failed: /home/pyb78/pyb78.specind.net/success.py, referer: http://pyb78.specind.net/option.html
[Wed Nov 20 15:30:21.993885 2019] [cgi:error] [pid 21996] [client 223.228.136.70:6088] End of script output before headers: success.py, referer: http://pyb78.specind.net/option.html
[Wed Nov 20 15:36:11.375845 2019] [cgi:error] [pid 22175] [client 223.228.136.70:58723] AH01215: (13)Permission denied: exec of '/home/pyb78/pyb78.specind.net/success.py' failed: /home/pyb78/pyb78.specind.net/success.py, referer: http://pyb78.specind.net/option.html
[Wed Nov 20 15:36:11.376892 2019] [cgi:error] [pid 22175] [client 223.228.136.70:58723] End of script output before headers: success.py, referer: http://pyb78.specind.net/option.html
[Wed Nov 20 15:58:15.179376 2019] [cgi:error] [pid 22968] [client 223.228.136.70:61948] AH01215:   File "/home/pyb78/pyb78.specind.net/success.py", line 56: /home/pyb78/pyb78.specind.net/success.py, referer: http://pyb78.specind.net/option.html
[Wed Nov 20 15:58:15.179455 2019] [cgi:error] [pid 22968] [client 223.228.136.70:61948] AH01215:     return response: /home/pyb78/pyb78.specind.net/success.py, referer: http://pyb78.specind.net/option.html
[Wed Nov 20 15:58:15.179477 2019] [cgi:error] [pid 22968] [client 223.228.136.70:61948] AH01215:     ^: /home/pyb78/pyb78.specind.net/success.py, referer: http://pyb78.specind.net/option.html
[Wed Nov 20 15:58:15.179509 2019] [cgi:error] [pid 22968] [client 223.228.136.70:61948] AH01215: SyntaxError: 'return' outside function: /home/pyb78/pyb78.specind.net/success.py, referer: http://pyb78.specind.net/option.html
[Wed Nov 20 15:58:15.181499 2019] [cgi:error] [pid 22968] [client 223.228.136.70:61948] End of script output before headers: success.py, referer: http://pyb78.specind.net/option.html
[Thu Nov 21 10:20:01.291983 2019] [autoindex:error] [pid 27010] [client 106.203.75.144:22596] AH01276: Cannot serve directory /home/pyb78/pyb78.specind.net/: No matching DirectoryIndex (index.py) found, and server-generated directory index forbidden by Options directive
[Thu Nov 21 19:43:44.558063 2019] [autoindex:error] [pid 16831] [client 111.92.72.169:35910] AH01276: Cannot serve directory /home/pyb78/pyb78.specind.net/: No matching DirectoryIndex (index.py) found, and server-generated directory index forbidden by Options directive
[Sat Nov 23 12:08:21.237304 2019] [php7:error] [pid 8867] [client 192.168.18.45:1890] script '/home/pyb78/pyb78.specind.net/action_page.php' not found or unable to stat, referer: http://pyb78.specind.net/signup.html
[Sat Nov 23 12:08:52.322005 2019] [php7:error] [pid 8893] [client 192.168.18.45:1891] script '/home/pyb78/pyb78.specind.net/action_page.php' not found or unable to stat, referer: http://pyb78.specind.net/signup.html
[Sat Nov 23 12:12:12.293888 2019] [php7:error] [pid 9028] [client 192.168.18.45:1892] script '/home/pyb78/pyb78.specind.net/action_page.php' not found or unable to stat, referer: http://pyb78.specind.net/signup.html
[Sat Nov 23 12:14:45.364984 2019] [php7:error] [pid 9104] [client 192.168.18.45:1903] script '/home/pyb78/pyb78.specind.net/action_page.php' not found or unable to stat
[Sat Nov 23 12:15:29.131836 2019] [php7:error] [pid 9136] [client 192.168.18.45:1907] script '/home/pyb78/pyb78.specind.net/action_page.php' not found or unable to stat
[Mon Nov 25 09:20:01.990962 2019] [autoindex:error] [pid 4405] [client 192.168.18.45:1162] AH01276: Cannot serve directory /home/pyb78/pyb78.specind.net/: No matching DirectoryIndex (index.py) found, and server-generated directory index forbidden by Options directive
[Mon Nov 25 09:20:06.884976 2019] [autoindex:error] [pid 4405] [client 192.168.18.45:1162] AH01276: Cannot serve directory /home/pyb78/pyb78.specind.net/: No matching DirectoryIndex (index.py) found, and server-generated directory index forbidden by Options directive
[Mon Nov 25 10:54:56.059229 2019] [php7:error] [pid 7552] [client 192.168.18.24:2150] script '/home/pyb78/pyb78.specind.net/action_page.php' not found or unable to stat, referer: http://pyb78.specind.net/signup.html
[Mon Nov 25 11:44:33.388880 2019] [autoindex:error] [pid 9170] [client 192.168.18.24:2426] AH01276: Cannot serve directory /home/pyb78/pyb78.specind.net/: No matching DirectoryIndex (index.py) found, and server-generated directory index forbidden by Options directive
[Mon Nov 25 15:24:35.009127 2019] [cgi:error] [pid 17225] [client 192.168.15.155:43226] AH01215: (26)Text file busy: exec of '/home/pyb78/pyb78.specind.net/login.py' failed: /home/pyb78/pyb78.specind.net/login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:24:35.010187 2019] [cgi:error] [pid 17225] [client 192.168.15.155:43226] End of script output before headers: login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:30:44.542338 2019] [cgi:error] [pid 17459] [client 192.168.15.155:43268] AH01215: (26)Text file busy: exec of '/home/pyb78/pyb78.specind.net/login.py' failed: /home/pyb78/pyb78.specind.net/login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:30:44.543357 2019] [cgi:error] [pid 17459] [client 192.168.15.155:43268] End of script output before headers: login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:32:59.057219 2019] [cgi:error] [pid 17551] [client 192.168.15.155:43274] AH01215: (26)Text file busy: exec of '/home/pyb78/pyb78.specind.net/login.py' failed: /home/pyb78/pyb78.specind.net/login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:32:59.058145 2019] [cgi:error] [pid 17551] [client 192.168.15.155:43274] End of script output before headers: login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:34:02.032839 2019] [cgi:error] [pid 17598] [client 192.168.15.155:43276] AH01215: (26)Text file busy: exec of '/home/pyb78/pyb78.specind.net/login.py' failed: /home/pyb78/pyb78.specind.net/login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:34:02.033845 2019] [cgi:error] [pid 17598] [client 192.168.15.155:43276] End of script output before headers: login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:34:47.226891 2019] [cgi:error] [pid 17625] [client 192.168.15.155:43282] AH01215: (26)Text file busy: exec of '/home/pyb78/pyb78.specind.net/login.py' failed: /home/pyb78/pyb78.specind.net/login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:34:47.227967 2019] [cgi:error] [pid 17625] [client 192.168.15.155:43282] End of script output before headers: login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:35:19.271094 2019] [cgi:error] [pid 17653] [client 192.168.15.155:43284] AH01215: (26)Text file busy: exec of '/home/pyb78/pyb78.specind.net/login.py' failed: /home/pyb78/pyb78.specind.net/login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:35:19.272163 2019] [cgi:error] [pid 17653] [client 192.168.15.155:43284] End of script output before headers: login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:36:15.323076 2019] [cgi:error] [pid 17685] [client 192.168.15.155:43286] AH01215: (26)Text file busy: exec of '/home/pyb78/pyb78.specind.net/login.py' failed: /home/pyb78/pyb78.specind.net/login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:36:15.324138 2019] [cgi:error] [pid 17685] [client 192.168.15.155:43286] End of script output before headers: login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:39:49.378334 2019] [cgi:error] [pid 17843] [client 192.168.15.155:43302] AH01215: (26)Text file busy: exec of '/home/pyb78/pyb78.specind.net/login.py' failed: /home/pyb78/pyb78.specind.net/login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:39:49.379395 2019] [cgi:error] [pid 17843] [client 192.168.15.155:43302] End of script output before headers: login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:41:09.529213 2019] [cgi:error] [pid 17877] [client 192.168.15.155:43332] AH01215: (26)Text file busy: exec of '/home/pyb78/pyb78.specind.net/login.py' failed: /home/pyb78/pyb78.specind.net/login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 15:41:09.530241 2019] [cgi:error] [pid 17877] [client 192.168.15.155:43332] End of script output before headers: login.py, referer: http://pyb78.specind.net/login.html
[Mon Nov 25 16:32:43.319261 2019] [cgi:error] [pid 19686] [client 223.228.128.89:44469] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 100: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:32:43.319370 2019] [cgi:error] [pid 19686] [client 223.228.128.89:44469] AH01215:     redirectURL = "http://pyb78.specind.net/login.html": /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:32:43.319390 2019] [cgi:error] [pid 19686] [client 223.228.128.89:44469] AH01215:               ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:32:43.319426 2019] [cgi:error] [pid 19686] [client 223.228.128.89:44469] AH01215: IndentationError: expected an indented block: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:32:43.321431 2019] [cgi:error] [pid 19686] [client 223.228.128.89:44469] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:34:47.823686 2019] [cgi:error] [pid 19747] [client 223.228.128.89:7792] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 100: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:34:47.823796 2019] [cgi:error] [pid 19747] [client 223.228.128.89:7792] AH01215:     redirectURL = "http://pyb78.specind.net/login.html": /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:34:47.823816 2019] [cgi:error] [pid 19747] [client 223.228.128.89:7792] AH01215:               ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:34:47.823851 2019] [cgi:error] [pid 19747] [client 223.228.128.89:7792] AH01215: IndentationError: expected an indented block: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:34:47.825662 2019] [cgi:error] [pid 19747] [client 223.228.128.89:7792] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:36:14.850019 2019] [cgi:error] [pid 19808] [client 192.168.15.155:43532] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 100: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:36:14.850116 2019] [cgi:error] [pid 19808] [client 192.168.15.155:43532] AH01215:     redirectURL = "http://pyb78.specind.net/login.html": /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:36:14.850132 2019] [cgi:error] [pid 19808] [client 192.168.15.155:43532] AH01215:               ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:36:14.850166 2019] [cgi:error] [pid 19808] [client 192.168.15.155:43532] AH01215: IndentationError: expected an indented block: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:36:14.851950 2019] [cgi:error] [pid 19808] [client 192.168.15.155:43532] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:36:59.946212 2019] [cgi:error] [pid 19812] [client 223.228.128.89:44748] AH01215:   File "/home/pyb78/pyb78.specind.net/register.py", line 100: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:36:59.946321 2019] [cgi:error] [pid 19812] [client 223.228.128.89:44748] AH01215:     redirectURL = "http://pyb78.specind.net/login.html": /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:36:59.946339 2019] [cgi:error] [pid 19812] [client 223.228.128.89:44748] AH01215:               ^: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:36:59.946375 2019] [cgi:error] [pid 19812] [client 223.228.128.89:44748] AH01215: IndentationError: expected an indented block: /home/pyb78/pyb78.specind.net/register.py, referer: http://pyb78.specind.net/registration.html
[Mon Nov 25 16:36:59.948291 2019] [cgi:error] [pid 19812] [client 223.228.128.89:44748] End of script output before headers: register.py, referer: http://pyb78.specind.net/registration.html
